sync/http/component_data_test.go

```diff
@@ -0,0 +1,68 @@
+package http
+
+import (
+	"time"
+
+	"github.com/beatlabs/patron/errors"
+)
+
+// Data used for testing the HTTP Component builder
+
+var httpBuilderNoErrors = []error{}
+var httpBuilderAllErrors = []error{
+	errors.New("Nil AliveCheckFunc was provided"),
+	errors.New("Nil ReadyCheckFunc provided"),
+	errors.New("Invalid HTTP Port provided"),
+	errors.New("Negative or zero read timeout provided"),
+	errors.New("Negative or zero write timeout provided"),
+	errors.New("Empty Routes slice provided"),
+	errors.New("Empty list of middlewares provided"),
+	errors.New("Invalid cert or key provided"),
+}
+
+type builderTestcase struct {
```

Comment:
Check out the std way of doing table driven tests...
e.g. https://github.com/taxibeat/rems/blob/develop/pkg/microrule/microrule_test.go

Reply:
Done. The `component_data_test.go` no longer exists; all data and tests have been merged into `component_test.go`